app/common/src/Device.js

```diff
@@ -19,4 +26,13 @@ export default {
     const { height, width } = renewDimensions();
     return Math.min(height, width);
   },
+  getDimensions() {
+    return renewDimensions();
```

Comment:
Do not expose this internal function. It renews dimensions every time you call it and it leads to inefficient usage like this (which is wrong):

```js
width: Device.getDimensions().width - SEARCH_MIN_WIDTH,
height: Device.getDimensions().height,
```